samples: nrf9160: modem_shell: link ncellmeas: interval option (#7004)
nrfconnect/sdk-nrf: doc/nrf/releases/release-notes-changelog.rst

  * Shell functionality to HTTP Update samples.

* Updated:

  * :ref:`modem_shell_application` sample:

    * Added:

>       * Addition to neighbor cell measurements in continuous mode (``link ncellmeass --continuous``): optional interval option ``--interval`` (in seconds) can be given and then new measurement is started in each interval seconds after previous measurement was reported.

Reviewer: ```suggestion
      * An option ``--interval`` (in seconds) to neighbor cell measurements in continuous mode  (``link ncellmeass --continuous``). 
        When using this option, a new measurement is started in each interval.
```

Author: Thanks! Changed accordingly.
